packages/quick_actions/quick_actions_ios/example/test_driver/integration_test.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart=2.9
```

Comment:
We should definitely be able to remove this.

Reply:
Removed.